### DISCUSSION TITLE
Recoil Barrel shots should explode on impact!

### DISCUSSION THREAD

**Discussion by pinesh** [tsunamayo/Starship-EVO#4320] [Suggestions]
Hiya!

Currently, it feels like the recoiling barrel, while immensely cool to use, doesn't distinguish itself enough from the range barrel to be chosen over it. The single-shot damage just doesn't add enough of an incentive with the reduction in speed and overall DPS.
![Alt Text](https://media.giphy.com/media/xUA7b7HcF126GuwoiA/giphy.gif)

_Recoiling!_

Much like artillery and other recoiling guns of old, perhaps the rounds should have some kind of splash damage on impact? 
Explosive shots could add enough flavor to make the barrels a serious contender.

I understand there are all sorts of overhead for explosive damage mechanics, but I imagine you'll cover it when missiles/rockets are re-introduced?

Thanks!
Harry

**Comment by SnowyBirdot** [comment]
As me, and lemons agree, it'd be awesome!
